### ISSUE TITLE
Unable to clean out Trash folder

### ISSUE THREAD

**Issue by VillageCatDad** [brief-rss/brief#430]
I went to look at the Trash folder out of curiosity, and I have items in there more than 10 years old!  When I right click, and select Empty Trash, nothing happens.  How do I clear out the Trash folder without having to go one-by-one and mark each article as "Read"?

**Comment by n8v8R** [comment]
right click on trash -> Empty Trash

**Comment by VillageCatDad** [comment]
Obviously, that didn't work at the time.  The problem was, most likely, a corrupted FF profile.
